raspios/overlay-rootfs/root/cogip.code-workspace

```diff
@@ -0,0 +1,20 @@
+{
+    "folders": [
+            {
+                    "path": "../boot"
+            },
+            {
+                    "path": "../etc"
+            },
+            {
+                    "path": "."
+            },
+            {
+                    "path": "../home/pi"
+            },
+            {
+                    "path": "../opt/venv/lib/python3.11/site-packages/cogip"
+            }
+    ],
+    "settings": {}
+}
```

Comment:
is it related to vscode ?

If yes, should not be committed. Or at least, even if ugly, in a separated commit.

Reply:
I thought it was already present and just moved from pi homedir, but you are right it 's a new file so should be in another commit.
But there is nothing ugly about this file, it is useful to have a preconfigured workspace to avoid to re-create each time.

Reply:
Done in another commit.